pkg/analytics/config.go

```diff
@@ -107,7 +110,9 @@ func get() *Analytics {
 		log.Debugf("error unmarshaling analytics: %s", err)
 		return &Analytics{Enabled: false, MachineID: ""}
 	}
-
+	if !okteto.GetTelemetryEnabled() {
```

Comment:
Maybe it is better to not initialize this here. And add and extra check if it is an okteto context and telemetry is disabled here:
https://github.com/okteto/okteto/blob/master/pkg/analytics/track.go#L286